### PR TITLE
Ofmcc 244 impersonate

### PR DESCRIPTION
**Pull request by jgstorey**

### CODE REVIEW

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/styles/common.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+.cc-top-level-card {
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/styles/common.css:1]: Confirming complete.

File: frontend/src/components/TheHeader.vue [hunk 2]
```diff
@@ -15,27 +15,26 @@
           </v-row>
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/components/TheHeader.vue:15]: The header user profile UI component dropdown (i.e. Home, Log Out) has been updated to include impersonate when a ministry user is signed in. This matches its accessible location in MyCSS. The bugs i was previous having around showing user profile icon have been resolved. If you still think we shouldn't show, I'll remove and re-add the 'Log Out' link and add an 'Impersonate Link'?

File: frontend/src/styles/common.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/styles/common.css:1]: image

File: frontend/src/views/HomeView.vue [hunk 3]
```diff
@@ -3,21 +3,27 @@
     <v-container>
       <v-card>
         <v-card-title>
-          User
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/views/HomeView.vue:6]: Are we sure its going to be that much of an issue for quick re-QA of small scree... screen is temporary.

File: frontend/src/views/LoginView.vue [hunk 4]
```diff
@@ -59,7 +66,28 @@
...
             <v-card-actions>
-              <AppButton icon="mdi-login" href="//www.bceid.ca/register/business/getting_started/getting_started.aspx" @click="clearStorage" width="250px">Register for a BCeID</AppButton>
+              <AppButton icon="mdi-login" href="//www.bceid.ca/register/business/getting_started/getting_started.aspx"
+                @click="clearStorage" width="250px">Register for a BCeID</AppButton>
+            </v-card-actions>
+          </v-card>
+        </v-col>
+
+
+        <v-col cols="4" class="xs-12 lg-6 xl-2">
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/views/LoginView.vue:76]: Confirming complete. i.e. http://localhost:8082/internal

File: frontend/src/views/ImpersonateView.vue [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                    this.setFailureAlert('An error occurred while trying to load BCeID')
+                }
+            }
+        }
+    }
+};
+</script>
+  
+<style scoped>
+.v-card {
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/views/ImpersonateView.vue:72]: Confirming fixed.

[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/views/ImpersonateView.vue:72]: Confirming fixed.

File: frontend/src/App.vue [hunk 6]
```diff
@@ -29,6 +30,7 @@ import { useAppStore } from '@/stores/app'
 import { useAuthStore } from '@/stores/auth'
 import HttpStatus from 'http-status-codes'
 
+
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/App.vue:33]: Confirming fixed.

File: frontend/src/App.vue [hunk 7]
```diff
@@ -81,7 +85,7 @@ export default {
   },
   methods: {
     ...mapActions(useAppStore, ['getConfig']),
-    ...mapActions(useAuthStore, ['getJwtToken']),
+    ...mapActions(useAuthStore, ['getJwtToken', 'logout']),
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/App.vue:88]: Confirmed fixed.

File: frontend/src/components/TheHeader.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/components/TheHeader.vue:15]: i.e
image

File: frontend/src/styles/common.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (weskubo-cgi) [reply frontend/src/styles/common.css:1]: Just leave it then.

File: frontend/src/views/HomeView.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (weskubo-cgi) [reply frontend/src/views/HomeView.vue:6]: Sure. It's temporary so let's just leave it.

File: frontend/src/components/TheHeader.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (weskubo-cgi) [reply frontend/src/components/TheHeader.vue:15]: We can leave then if it helps with navigation/etc. We don't have a formal design anyways.

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**